vispy/visuals/bar.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+import numpy as np
+from .mesh import MeshVisual
+
+
+class BarVisual(MeshVisual):
+    """Visual that calculates and displays a bar graph
+
+    Parameters
+    ----------
+    data : array-like
+        1st column is the height, optional 2nd column is bottom
+    height : array-like
+        Height of the bars
+    bottom : array-like
+        Bottom of the bars
+    width : int | float
+        Width of all bars
+    shift : int | float
+        Shift of all bars along the x-axis
+    color : instance of Color
+        Color of the bar.
+    orientation : {'h', 'v'}
+        Orientation of the histogram - 'h' is default
+    """
+
+    def __init__(self, height, bottom=None, width=0.8, shift=0.0, color='w', orientation='h'):
```

Comment:
Would be awesome to be able to pass a per-bar color as well!

Reply:
I think I implemented something kinda like that in #2388 and those PR's are very samey.
#2388 has also more comments trying to explain whats going on

Reply:
Yeah no need to do this here, I just pointed out it would be cool to add in the future :)

Reply:
Well I just looked into it, it should be not to problematic